spatialdata/_core/_rasterize.py

```diff
@@ -395,25 +414,36 @@ def _(
         min_coordinate=min_coordinate,
         max_coordinate=max_coordinate,
         target_sizes=target_sizes,
-        corrected_affine=corrected_affine,
+        target_coordinate_system=target_coordinate_system,
+        # corrected_affine=corrected_affine,
     )
 
     if pyramid_scale is not None:
         extra = [pyramid_scale.inverse()]
     else:
         extra = []
 
+    # get inverse transformation
+    corrected_affine, target_axes = _get_corrected_affine_matrix(
+        data=data,
+        axes=axes,
+        min_coordinate=min_coordinate,
+        max_coordinate=max_coordinate,
+        target_coordinate_system=target_coordinate_system,
+    )
+
     half_pixel_offset = Translation([0.5, 0.5, 0.5], axes=("z", "y", "x"))
     sequence = Sequence(
         [
-            half_pixel_offset.inverse(),
+            # half_pixel_offset.inverse(),
             scale,
             translation,
             corrected_affine.inverse(),
-            half_pixel_offset,
+            # half_pixel_offset,
```

Comment:
does this need to be commented out or can be removed?

Reply:
I would keep it commented until this is addressed: https://github.com/scverse/spatialdata/issues/165

which we should do with medium priority